custom_components/isy994/sensor.py

```diff
@@ -49,6 +48,7 @@ async def async_setup_entry(
 
     await migrate_old_unique_ids(hass, DOMAIN, devices)
     async_add_entities(devices)
+    await async_setup_device_services(hass)
```

Comment:
```suggestion
   async_setup_device_services(hass)
```

These service setup functions aren't awaiting anything so they could be normal functions decorated with `@callback` instead of async functions.